modules/nextflow/src/main/groovy/nextflow/executor/BridgeExecutor.groovy

```diff
@@ -0,0 +1,179 @@
+     *
+     * @param task A {@link TaskRun} to be submitted
+     * @param result The {@link List} instance to which add the job directives
+     * @return A {@link List} containing all directive tokens and values.
+     */
+    protected List<String> getDirectives(TaskRun task, List<String> result) {
+
+        String job_name = ""
+        // parenthesis are not compatible with bridge submission commands
+        job_name = getJobNameFor(task)
+        job_name = job_name.replace("(", "")
+        job_name = job_name.replace(")", "")
+
+        result << '-r' << job_name 
+        result << '-o' << quote(task.workDir.resolve(TaskRun.CMD_LOG)) 
+
+        // number of cores per parallel task to allocate 
+        if( task.config.cpus > 1 ) {
+            result << '-c' << task.config.cpus.toString()
+        }
+
+        // maximum walltime of the batch job in seconds
+        if( task.config.time ) {
+            result << '-T' << task.config.getTime().toSeconds() 
+        }
+
+        // maximum memory amount required per allocated core in Mo (default is chosen by the underlying system)
+        if( task.config.getMemory() ) {
+            result << '-M' << task.config.getMemory().toMega().toString() 
+        }
+
+        // the requested partition (a.k.a queue) name
+        if( task.config.queue ) {
+            result << '-q' << (task.config.queue.toString())
+        }
+
+        // other cluster options 
+        if( task.config.clusterOptions ) {
+            result << task.config.clusterOptions.toString() << ''
+        }
+
+        return result
+    }
+
+    String getHeaderToken() { '#MSUB' }
+
+    /**
+     * The command line to submit this job
+     *
+     * @param task The {@link TaskRun} instance to submit for execution to the cluster
+     * @param scriptFile The file containing the job launcher script
+     * @return A list representing the submit command line
+     */
+    @Override
+    List<String> getSubmitCommandLine(TaskRun task, Path scriptFile ) {
+
+        ['ccc_msub', scriptFile.getName()]
+
+    }
+
+    /**
+     * Parse the string returned by the {@code sbatch} command and extract the job ID string
+     *
+     * @param text The string returned when submitting the job
+     * @return The actual job ID string
+     */
+    @Override
+    def parseJobId(String text) {
+
+        for( String line : text.readLines() ) {
+            def m = SUBMIT_REGEX.matcher(line)
+            if( m.find() ) {
+                return m.group(1).toString()
+            }
+        }
+
+        def id = text.trim()
+        if( id.isLong() )
+            return id
+
+        throw new IllegalStateException("Invalid BRIDGE submit response:\n$text\n\n")
+    }
+
+    @Override
+    protected List<String> getKillCommand() { ['ccc_mdel'] }
+
+    @Override
+    protected List<String> queueStatusCommand(Object queue) {
+
+        final result = ['squeue','--noheader','-o','%i %t', '-t', 'all']
```

Comment:
Is this command correct? Looks like copy-paste from SLURM.

Reply:
Indeed. Now changed to bridge command in latest commit.